charts/pega/README.md

```diff
@@ -801,6 +801,32 @@ pegasearch:
   externalURL: "http://srs-service.namespace.svc.cluster.local"
 ```
 
+To use OAuth authorization service for the connection between Pega Infinity and Search and Reporting Service, follow the instructions provided at ['backingservices'](../backingservices) to configure connection to the authorization service in Search and Reporting Service. You must configure authorization parameters for your Pega Infinity deployment using the parameter in values.yaml as shown in the following table and example:
+
+| Parameter             | Description                                                                                                                                                                                                 | Default value        |
+|-----------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------|
+| `enabled`             | Set the `pegasearch.srsAuth.enabled` as true to use OAuth between Infinity and Search and Reporting service                                                                                                 | false                |
+| `url`                 | Set the `pegasearch.srsAuth.url` value to URL of the OAuth endpoint to get the token for Search and Reporting service                                                                                       | `""`                 |
```

Comment:
Set the `pegasearch.srsAuth.url` value to the URL of the OAuth service endpoint to get the token for SRS.

Reply:
Done